android/src/main/java/com/imagepicker/Utils.java

```diff
@@ -478,14 +479,19 @@ static ReadableMap getImageResponseMap(Uri uri, Options options, Context context
       }
     }
 
-    static ReadableMap getVideoResponseMap(Uri uri, Context context) {
+    static ReadableMap getVideoResponseMap(Uri uri, Options options, Context context) {
         String fileName = uri.getLastPathSegment();
         WritableMap map = Arguments.createMap();
         map.putString("uri", uri.toString());
         map.putDouble("fileSize", getFileSize(uri, context));
         map.putInt("duration", getDuration(uri, context));
         map.putString("fileName", fileName);
         map.putString("type", getMimeType(uri, context));
+
+        if(options.includeExtra) {
+          map.putString("id", fileName);
```

Comment:
So the `includeExtra` flag is specifically for values which need permissions, does `id` need permissions to be accessed on iOS? I'm not sure if it's necessary for the `options.includeExtra` check here. Thoughts?

Reply:
@lukebrandonfarrell yeah, no permission is needed for Android. I was just trying to be consistent with iOS? Does fetching the timestamp on Android require permissions?

Can easily remove it, and add a note on the readme to say flag is only needed for iOS

Reply:
@helenaford fetching the timestamp doesn't require permission on Android, it's a hard decision because consistency simplifies the understanding of how the API works to consumers, but we could technically remove it on Android. I say let's keep it for now and I can remove it in my next PR for Android, as I will be adding `datetime` to video next week.